pennylane_lightning/lightning_qubit/lightning_qubit2.py

```diff
@@ -194,7 +201,13 @@ class LightningQubit2(Device):
 
     _device_options = ("rng", "c_dtype", "batch_obs", "mcmc", "kernel_name", "num_burnin")
 
+    _new_API = True
     _CPP_BINARY_AVAILABLE = LQ_CPP_BINARY_AVAILABLE
+    short_name = "lightning.qubit2"
+    operations = _operations
+    observables = _observables
```

Comment:
So we do have these as class properties below as well?

Reply:
I think the duplication was introduced during a merge. Let me keep only these two.